alvr/client/src/connection.rs

```diff
@@ -214,7 +215,7 @@ async fn connection_pipeline(
 
     let version = Version::from_str(&config_packet.reserved).ok();
     if version
-        .map(|v| v >= Version::from((15, 1, 0)))
+        .map(|v| v >= Version::from_str("15.1.0").unwrap())
```

Comment:
You could also use [built](https://lib.rs/crates/built).
```suggestion
        .map(|v| v >= Version::from_str(env!("CARGO_PKG_VERSION")).unwrap())
```

Reply:
I would avoid adding crates if not strictly necessary.
Your suggested change is not correct. We need the version specified.

Reply:
Ah, backwards compatibility - not the latest version. (oops)